module/observations/helper.js

```diff
@@ -1048,4 +1052,179 @@ module.exports = class ObservationsHelper {
         });
     }
 
+
+         /**
+      * observation link.
+      * @method
+      * @name getObservationLink
+      * @param  {String} observationSolutionId observation solution external Id.
+      * @param  {String} appName name of app.
+      * @returns {getObservationLink} observation getObservationLink.
+     */
+
+    static getObservationLink(observationSolutionId, appName) {
+        return new Promise(async (resolve, reject) => {
+            try {
+
+                if (!observationSolutionId || observationSolutionId == "" || !appName || appName == "") {
+                    throw messageConstants.apiResponses.INVALID_PARAMETER;
+                }
+
+                let link = null;
+                
+                let observationData = await solutionHelper.solutionDocuments({
+                        externalId : observationSolutionId,
+                        isReusable : false,
+                        type : messageConstants.common.OBSERVATION
+
+                        },[
+                            "link"
+                    ]);
+
+                if(!observationData || observationData == "") {
+                    return resolve({
+                        message: messageConstants.apiResponses.OBSERVATION_NOT_FOUND,
+                        result: {}
+                    });
+                }
+
+                let appDetails = await kendraService.getAppDetails(appName);
+                
+                if(appDetails.result === false){
```

Comment:
@priyanka-TL  Do not throw error if api is used by frontend instead throw return resolve({message : "App details not found", result : {}})